notebooks/niriss_ami_binary/3_niriss_ami_binary.ipynb

```diff
@@ -0,0 +1,751 @@
+{
```

Comment:
<ol><li>The need to manually add a forward slash to <code>datadir</code> for this to work is more clunky than I'd like.</li><li>I noticed that <code>fouriever</code> only accepts string paths, which I assume is the reason behind number 1. Accepting <code>pathlib.Path</code> objects could make things more flexible internally on the <code>fouriever</code> side and fit better with the approaches to file handling taken in more recently reviewed JDAT notebooks.</li><li>This cell creates an extra, unneeded file for me in the <code>niriss_ami_binary</code> directory : <code>niriss_ami_binary3calib_abdor_f480m_pos1_hd37093.oifits</code>. I'm guessing some part of <code>fouriever</code>'s code is missing the slash that should go between <code>niriss_ami_binary3</code> and the rest of the string. Again, maybe adopting the approach suggested in number 2 could prevent this.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/100/discussion/#comment-1116103364'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>